hack/test_module.sh

```diff
@@ -0,0 +1,235 @@
+elif [ $kubernetesVersion == "kind24" ]
+then
+    ${TOOLBIN}/kind delete cluster
+    ${TOOLBIN}/kind create cluster --image=kindest/node:v1.24.7@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
+elif [ $kubernetesVersion == "kind25" ]
+then
+    ${TOOLBIN}/kind delete cluster
+    ${TOOLBIN}/kind create cluster --image=kindest/node:v1.25.3@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1
+else
+    echo "Unsupported kind version"
+    exit 1
+fi
+
+
+# starting with write operation
+
+${TOOLBIN}/helm repo add jetstack https://charts.jetstack.io
+${TOOLBIN}/helm repo add hashicorp https://helm.releases.hashicorp.com
+${TOOLBIN}/helm repo add fybrik-charts https://fybrik.github.io/charts
+${TOOLBIN}/helm repo update
+
+
+${TOOLBIN}/helm install cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --version v$certManagerVersion \
+    --create-namespace \
+    --set installCRDs=true \
+    --wait --timeout 400s
+
+if [ $fybrikVersion == "master" ]
+then
+	rm -rf fybrik
+	git clone https://github.com/fybrik/fybrik.git
+	cd fybrik
+	../${TOOLBIN}/helm dependency update charts/vault
+	../${TOOLBIN}/helm install vault charts/vault --create-namespace -n fybrik-system \
+	    --set "vault.injector.enabled=false" \
+	    --set "vault.server.dev.enabled=true" \
+	    --values charts/vault/env/dev/vault-single-cluster-values.yaml
+	../${TOOLBIN}/kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=120s
+	../${TOOLBIN}/helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
+	../${TOOLBIN}/helm install fybrik charts/fybrik --set "coordinator.catalog=katalog" --set global.tag=master -n fybrik-system --wait
+	cd -
+	rm -rf fybrik
+else
+	${TOOLBIN}/helm install vault fybrik-charts/vault --create-namespace -n fybrik-system \
+        --set "vault.injector.enabled=false" \
+        --set "vault.server.dev.enabled=true" \
+        --values https://raw.githubusercontent.com/fybrik/fybrik/v$fybrikVersion/charts/vault/env/dev/vault-single-cluster-values.yaml
+    ${TOOLBIN}/kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=400s
+
+	${TOOLBIN}/helm install fybrik-crd fybrik-charts/fybrik-crd -n fybrik-system --version v$fybrikVersion --wait
+	${TOOLBIN}/helm install fybrik fybrik-charts/fybrik --set "coordinator.catalog=katalog" -n fybrik-system --version v$fybrikVersion --wait
+fi
+
+# apply modules
+
+# Related to https://github.com/cert-manager/cert-manager/issues/2908
+# Fybrik webhook not really ready after "helm install --wait"
+# A workaround is to loop until the module is applied as expected
+if [ $moduleVersion == "master" ]
+then
+	CMD="${TOOLBIN}/kubectl apply -f ../module.yaml -n fybrik-system"
+else
+	CMD="${TOOLBIN}/kubectl apply -f https://github.com/fybrik/arrow-flight-module/releases/download/v$moduleVersion/module.yaml -n fybrik-system"
```

Comment:
should this be the `airbyte-module` rather than the `arrow-flight-module`?

Reply:
Fixed thanks